doc/classes/Gradient.xml

```diff
@@ -97,5 +100,14 @@
 		<constant name="GRADIENT_INTERPOLATE_CUBIC" value="2" enum="InterpolationMode">
 			Cubic interpolation.
 		</constant>
+		<constant name="GRADIENT_COLOR_SPACE_SRGB" value="0" enum="ColorSpace">
+			SRGB color space.
```

Comment:
Just a small nitpick, but I think we should use the standard notation for consistency, which starts with a lower case s.
```suggestion
			sRGB color space.
```